testsuite/databases/redis/service.py

```diff
@@ -123,3 +144,8 @@ def _resolve_hostname(hostname: str) -> str:
             return result[0][4][0]
     warnings.warn(f'Failed to resolve hostname {hostname}')
     return hostname
+
+def _get_service_script_path(cluster_mode: bool) -> pathlib.Path:
```

Comment:
you have to register redis service and redis-cluster service at the same time. Fixture must determinate which one must be used (ensure_service_started('redis' or 'redis-cluster'))

* redis and redis-cluster services must support simultaneous start
* which one service to use must be chosen with fixture
* redis and redis-cluster must use different ports
* please add tests for redis-cluster, use fixture to switch redis plugin to cluster instance instead of sentinel